style/style.less

```diff
@@ -0,0 +1,14 @@
+.wp-style {
+  background-color: #eaf3ff;
+  padding: 2px 2px 2px 2px;
+  font-family: 'Linux Libertine','Georgia','Times',serif;
+  border-radius: 4px;
+  border: solid 1px rgba(51, 102, 204, 0.48);
+  cursor: pointer;
+
+  &:after {
+    content: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='20' height='20' viewBox='0 0 20 20'><g fill='none' fill-rule='evenodd'><path d='M0 0L20 0 20 20 0 20z'/><path fill='%2336C' fill-opacity='.7' fill-rule='nonzero' d='M10.556 1.111c-4.6 0-8.334 3.733-8.334 8.333s3.734 8.334 8.334 8.334 8.333-3.734 8.333-8.334-3.733-8.333-8.333-8.333zm.833 12.5H9.722v-5h1.667v5zm0-6.667H9.722V5.278h1.667v1.666z'/></g></svg>");
```

Comment:
is there any loader so we can import the svg image here, then loader convert it to this format?

_I also need this loader in the layout ticket_

Reply:
Check out [this thread](https://github.com/wikimedia/wikipedia-preview/pull/5#discussion_r439390548), I tried with this plugin but ran into issues: https://github.com/atlassian/less-plugin-inline-svg

Reply:
Thanks! it works for me, checkout this commit
https://github.com/wikimedia/wikipedia-preview/pull/12/commits/64ee96d410b48abbd074259355f7028f2c5bc695